src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int STACK_VALUE = -1;
+
+	private static final int STACK_MAX_SIZE = 12;
+
+	private final int[] numbers = new int[STACK_MAX_SIZE];
+
+	private int total = STACK_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)